src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -164,65 +165,59 @@ private static void updateTrackingInfo(String statusType,
                                            DdpKit kit,
                                            Instant earliestInTransitTime) {
         String upsUpdate = statusType + " " + statusDescription;
-        SimpleResult result = inTransaction((conn) -> {
```

Comment:
Because this runs in its own transaction, if there is an error outside this block, the ups status might be updated--potentially to `D` or `I`--which will then result in subsequent calls to `shouldTriggerEventForKitOnItsWayToParticipant()` or `shouldTriggerEventForReturnKitDelivery()` returning `false` when they should return `true` because a subsequent run of the job will find a new value for `oldType`.  So